R/tdata.R

```diff
@@ -94,12 +94,22 @@ get_code.tdata <- function(data) {
   attr(data, "code")()
 }
 
+#' Wrapper for `get_code.tdata`
+#' This wrapper is to be used by downstream packages to extract the code of a `tdata` object
+#'
+#' @param data (`tdata`) object
+#'
+#' @return (`character`) code used in the `tdata` object.
+#' @export
+get_code_tdata <- function(data) {
+  get_code.tdata(data)
```

Comment:
So I think this should be:

```suggestion
  checkmate::assert_class(data, "tdata")
  get_code(data)
```